<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEditFolderPath">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>671</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>A:\Broomfield\ZDone</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonPickFolder">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>20</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Pick Root Folder</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditNameSubstring">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>220</y>
      <width>271</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>es, ky, lb, lm, jc</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelText4">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>250</y>
      <width>241</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Initials to look for (comma separated)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonStartNew">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>90</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Counting</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>320</y>
      <width>421</width>
      <height>221</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Results</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>381</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Pick the ZDone folder. Should look like: A:BroomfieldZDone</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>120</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>This might take a couple minutes</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
